Magefile.go

```diff
@@ -103,10 +106,25 @@ func Format() error {
 	return nil
 }
 
-// Dev starts a front-end development server.
+// Dev builds the plugin in dev mode
```

Comment:
Documentation comments should [end with a period](https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences).
```suggestion
// Dev builds the plugin in dev mode.
```